src/openloco/localisation/string_ids.h

```diff
@@ -612,7 +624,44 @@ namespace openloco::string_ids
     constexpr string_id industry_size_high = 1669;
     constexpr string_id number_of_industries = 1670;
 
+    constexpr string_id tooltip_scenario_options = 1674;
+    constexpr string_id tooltip_scenario_challenge = 1675;
+    constexpr string_id tooltip_company_options = 1676;
+    constexpr string_id tooltip_financial_options = 1677;
+    constexpr string_id title_scenario_options = 1678;
+    constexpr string_id title_scenario_challenge = 1679;
+    constexpr string_id title_company_options = 1680;
+    constexpr string_id title_financial_options = 1681;
+    constexpr string_id max_competing_companies = 1682;
+    constexpr string_id max_competing_companies_value = 1683;
+    constexpr string_id delay_before_competing_companies_start = 1684;
+    constexpr string_id delay_before_competing_companies_start_months = 1685;
+    constexpr string_id selection_of_competing_companies = 1686;
+    constexpr string_id preferred_intelligence = 1687;
+    constexpr string_id preferred_aggressiveness = 1688;
+    constexpr string_id preferred_competitiveness = 1689;
+    constexpr string_id preference_any = 1690;
+    constexpr string_id preference_low = 1691;
+    constexpr string_id preference_medium = 1692;
+    constexpr string_id preference_high = 1693;
+
```

Comment:
Why the blank line? The numbers are sequential

Reply:
They probably weren't when either of the strings were introduced. I'll amend.